### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by Spindel** [kevin-civic/civic_news#5]
Two sections are missing from the README

- How do you interact with it  
- How do you develop on it

**Comment by kevin-civic** [comment]
1. Is this in terms of user interactions?

2. What exactly does this mean "how do i develop on it?"

**Comment by Spindel** [comment]
## How do you develop on it?
So. Say you're someone who comes here from Reddit or somewhere and want to help out.

What should they be doing? How do they get the environment up, dependencies installed, and can start making a change?

That's what you should document, and keep up to date. For your own sake ( imagine getting a new machine)  and others.

## How do you interact with it?  
This is about administrators and end users, mostly end users. And require you to do a bit of planning.
How do you intend for users to _use_ this software.   Not "why", but "how".

**Comment by Spindel** [comment]
To link issues and commits, you mention the Issue in the tags part of the commit message ( bottom part)

````
Update readme with development info

For us to have new developers, we need to document how to set up a development environment, how to run testcases, and how to start out.   This commit resolves that.

Fixes issue #3
```
